src/mime.tsx

```diff
@@ -120,8 +121,9 @@ export class RenderedMySTMarkdown
    *
    * @returns A promise which resolves when rendering is complete.
    */
-  renderModel(model: IRenderMime.IMimeModel): Promise<void> {
-    const mdast = markdownParse(model.data[MIME_TYPE] as string);
+  async renderModel(model: IRenderMime.IMimeModel): Promise<void> {
+    const markdownText = model.data[MIME_TYPE] as string;
+    const mdast = markdownParse(markdownText, false);
```

Comment:
@agoose77 -- really weird that the `markdownText` never actually contains the yaml frontmatter block!? Would love your help debugging that...